src/main/java/hudson/plugins/ws_cleanup/WsCleanup.java

```diff
@@ -136,6 +187,9 @@ public String getExternalDelete() {
     }
 
     private boolean shouldCleanBuildBasedOnState(Result result) {
+        if (result == null)
+            // in case of Pipeline, the result may be null
+            return true;
```

Comment:
I do agree for using brackets even for single-line conditions, but in this case I though to be consistent with the existing code. The next if statements don't have brackets. Should I also refactor the next lines within this method?